lib/driver.js

```diff
@@ -395,13 +395,14 @@ class XCUITestDriver extends BaseDriver {
           this.logEvent('customCertInstalled');
         }
       }
-
-      try {
-        const idb = new IDB({udid});
-        await idb.connect();
-        this.opts.device.idb = idb;
-      } catch (e) {
-        log.info(`idb will not be used for Simulator interaction. Original error: ${e.message}`);
+      if (this.opts.launchWithIDB && this.isSimulator()) {
```

Comment:
this condition should not be applied here

We are anyway currently in the `if` block for simulator only and we agreed that idb could be used for other simulator purposes, not only for app launch, so it makes sense to always try initing it